1.9/usage/cli/command-reference/dcos-marathon/dcos-marathon-app-add.md

```diff
@@ -0,0 +1,77 @@
+---
+post_title: dcos marathon app add
+menu_order: 1
+---
+
+# Description
+Add an application.
+
+# Usage
+
+```bash
+dcos marathon app add <app-resource> [OPTION]
+```
+
+# Options
+
+None.
+
+# Positional arguments
+
+| Name, shorthand | Default | Description |
+|---------|-------------|-------------|
+| `<app-resource>`   |             |  Path to a file or HTTP(S) URL that contains the app's JSON definition. If omitted, the definition is read from stdin. For a detailed description see the [documentation](https://docs.mesosphere.com/usage/marathon/rest-api/). |
+
+# Parent command
+
+| Command | Description |
+|---------|-------------|
+| [dcos marathon](/docs/1.9/usage/cli/command-reference/dcos-marathon/) | Deploy and manage applications to DC/OS. |
+
+# Examples
+
+## Deploy a simple app
+
+In this example, a simple app is deployed to DC/OS Marathon.
+
+1.  Create an app definition file named `my-app.json` with these contents.
+
+    ```bash
+    {
+        "id": "/my-app",
+        "container": {
+        "type": "DOCKER",
+        "docker": {
+              "image": "group/image",
+              "network": "BRIDGE",
+              "portMappings": [
+                { "hostPort": 80, "containerPort": 80, "protocol": "tcp"}
+              ]
+            }
+        },
+        "instances": 1,
+        "cpus": 0.1,
+        "mem": 64
+    }
+    ```
+    
+1.  Add your app to Marathon:
+
+    ```bash
+    dcos marathon app add myapp.json
```

Comment:
variables should have greater than/less than signs around them, so `<myapp.json>`